wallet/src/actors/app/methods.rs

```diff
@@ -818,4 +829,141 @@ impl App {
 
         Box::new(f)
     }
+
+    /// Export wallet private key, encrypted with password
```

Comment:
I like the term "master key" (or something similar) to make it clearer that the exported key is the master (root) private key used to derive the addresses key pairs.